db/migrate/20210911041706_add_wallet_id_to_admins.rb

```diff
@@ -0,0 +1,5 @@
+class AddWalletIdToAdmins < ActiveRecord::Migration[6.0]
+  def change
+    add_column :admins, :wallet_id, :integer
```

Comment:
Why do we nedd to add `wallet_id` to the `admins` table?

Reply:
We deleted the migration, got confused with our ERD. `wallet_id` is not necessary. Thanks for pointing that out. :)